landing_page/app/views/microposts/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<p id="notice"><%= notice %></p>
```

Comment:
You have been using only `<p>` here, which isn't recommended as per HTML standards. Have a look at these:

- [HTML Sections](https://html.spec.whatwg.org/multipage/sections.html#sections), and how they should be used to categorize content
- [HTML Grouping Content](https://html.spec.whatwg.org/multipage/grouping-content.html#grouping-content)
- [HTML Text Level Semantics](https://html.spec.whatwg.org/multipage/text-level-semantics.html#text-level-semantics)